Phonebook.Frontend/src/app/shared/dialogs/user-information/incorrect-user-information.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="">
+  <h2 mat-dialog-title i18n="User-InformationDialog|Headline@@Headline">Incorrect User Information</h2>
+  <div mat-dialog-content *ngIf="onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Your Profile">
+      Dear {{ data.person.Firstname }},<br />
+      unfortunately, you can't change your user information via the Phonebook interface. <br />
+      If your data is incorrect, please contact your Human Resources department.
+    </span>
+  </div>
+  <div mat-dialog-content *ngIf="!onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Colleague Profile">
+      Is there something wrong with the profile? <br />
+      Please let your colleague know that the data is incorrect and that it can be changed through your Human Resources
+      department.
+    </span>
+  </div>
+  <div *ngIf="!onMyProfile">
+    <a
+      mat-icon-button
```

Comment:
Thats not an icon button, but a normal one 
```suggestion
      mat-raised-button
```